<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>benchmarkWindow</class>
 <widget class="QWidget" name="benchmarkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>830</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>← Go back</string>
   </property>
  </widget>
  <widget class="QLabel" name="pageName_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1501</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: orange;
font-size: 35px;
font-family: Roboto;</string>
   </property>
   <property name="text">
    <string>Benchmark</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="testAllButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>810</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Test all</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="arrayTable">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>155</y>
     <width>442</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #DE6262,  stop:1 #FFB88C);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
	font-size: 14px;
}

QTableView {
    selection-background-color: #dedede;

	selection-color: dark-grey;

	font-size:14px;
}


/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}

QTableView::item {
}
</string>
   </property>
   <column>
    <property name="text">
     <string>N</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Start N</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Create</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Insert</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Get</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Set</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Remove</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time Σ</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>105</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 28px;
font-weight: 500;
text-decoration: underline;
color: #e66739;</string>
   </property>
   <property name="text">
    <string>Vector</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testSpecStructButton">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>735</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NCounter">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>810</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000000</number>
   </property>
   <property name="singleStep">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>105</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 28px;
font-weight: 500;
text-decoration: underline;
color: #e66739;</string>
   </property>
   <property name="text">
    <string>Array</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testArrayButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>735</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="specStructTable">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>155</y>
     <width>442</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #DE6262,  stop:1 #FFB88C);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QTableView {
    selection-background-color: #dedede;

	selection-color: dark-grey;

	font-size:15px;
}


/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}

QTableView::item {
}
</string>
   </property>
   <column>
    <property name="text">
     <string>N</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Start N</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Create</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Insert</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Get</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Set</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Remove</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time Σ</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="vectorTable">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>155</y>
     <width>442</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #DE6262,  stop:1 #FFB88C);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QTableView {
    selection-background-color: #dedede;

	selection-color: dark-grey;

	font-size:15px;
}


/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}

QTableView::item {
}
</string>
   </property>
   <column>
    <property name="text">
     <string>N</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Start N</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Create</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Insert</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Get</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Set</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Remove</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time Σ</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>105</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 28px;
font-weight: 500;
text-decoration: underline;
color: #e66739;</string>
   </property>
   <property name="text">
    <string>Special structure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testVectorButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>735</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>810</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px;
font-weight: 500;</string>
   </property>
   <property name="text">
    <string>N =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>860</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
font-weight: 500;</string>
   </property>
   <property name="text">
    <string>Start elements  =</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="startNCounter">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>860</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="singleStep">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="errorMessage">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>860</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: red;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="message">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>860</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: green;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
